sci-physics/geant/geant-4.10.03.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils versionator multilib
+
+MY_P=${PN}${PV}
+SPV="$(get_version_component_range 1 ${PV}).$(get_version_component_range 2 ${PV}).$(printf %1d $(get_version_component_range 3 ${PV}))"
+
+DESCRIPTION="Toolkit for simulation of passage of particles through matter"
+HOMEPAGE="http://geant4.cern.ch/"
+SRC_URI="http://geant4.cern.ch/support/source/${MY_P}.tar.gz"
+
+LICENSE="geant4"
+SLOT="4"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+data dawn doc examples gdml geant3 inventor motif opengl
+	raytracerx qt4 static-libs vrml zlib threads"
+
+RDEPEND="
+	dev-libs/expat
+	>=sci-physics/clhep-2.3.3.0:2=
+	dawn? ( media-gfx/dawn )
+	gdml? ( dev-libs/xerces-c )
+	motif? ( x11-libs/motif:0 )
+	opengl? ( virtual/opengl )
+	inventor? ( media-libs/SoXt )
+	qt4? ( dev-qt/qtgui:4 opengl? ( dev-qt/qtopengl:4 ) )
+	raytracerx? ( x11-libs/libX11 x11-libs/libXmu )
+	zlib? ( sys-libs/zlib )"
+DEPEND="${RDEPEND} prefix? ( app-shells/tcsh )"
+PDEPEND="
+	data? ( ~sci-physics/geant-data-${PV} )
+	doc? ( ~app-doc/geant-docs-${PV} )"
+
+DOCS=( ReleaseNotes/ReleaseNotes${SPV}.html )
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	use examples || sed -i '/install(DIRECTORY examples/,/)/d' CMakeLists.txt || die
+	default
```

Comment:
cmake-utils -> never use `default`, but `cmake-utils_src_prepare` (and call it first in `src_prepare`)